test/set.js

```diff
@@ -11,23 +11,93 @@ describe('Set functions', function() {
   describe('#.intersection', function() {
 
     it('should properly compute the intersection of two sets.', function() {
-      var A = new Set([1, 2, 3]),
-          B = new Set([2, 3, 4]);
+      var Exemplar = new Set([2, 3, 1]);
+      var Equiv    = new Set([1, 2, 3]);
+      var Missing1 = new Set([2, 3]);
+      var Extra45  = new Set([1, 2, 3, 4, 5]);
+      var Disjoint = new Set([7, 8, 9]);
+      var Empty    = new Set([])
+      //
+      var WithItself   = functions.intersection(Exemplar, Exemplar);
+      assert.deepStrictEqual(Array.from(WithItself), [2, 3, 1]);
+      var WithEquiv    = functions.intersection(Exemplar, Equiv);
+      assert.deepStrictEqual(Array.from(WithEquiv), [2, 3, 1]);
+      var WithSubset   = functions.intersection(Exemplar, Missing1);
+      assert.deepStrictEqual(Array.from(WithSubset), [2, 3]);
+      var WithExtras   = functions.intersection(Exemplar, Extra45);
+      assert.deepStrictEqual(Array.from(WithExtras), [2, 3, 1]);
+      var WithDisjoint = functions.intersection(Exemplar, Disjoint);
+      assert.deepStrictEqual(Array.from(WithDisjoint), []);
+      var WithEmpty    = functions.intersection(Exemplar, Empty);
+      assert.deepStrictEqual(Array.from(WithEmpty), []);
+    });
 
-      var I = functions.intersection(A, B);
+    it('Makes no guarantees for element ordering.', function() {
```

Comment:
drop the capitalized `M` :)

Reply:
Fixed. In an act of overkill, because the jest eslinter enforces lowercase titles, I added the mocha linter to package.json. It does not; but it did catch some cases where there were arrow functions, which mocha doesn't like. I can remove or file that commit under its own PR if you prefer.